src/Filter/CompareFilter.php

```diff
@@ -4,26 +4,23 @@
 
 namespace Yiisoft\Data\Db\Filter;
 
-use InvalidArgumentException;
+use DateTimeInterface;
+use Yiisoft\Data\Db\ColumnFormatterTrait;
 use Yiisoft\Data\Reader\Filter\FilterInterface;
-use Yiisoft\Db\Expression\ExpressionInterface;
 
 abstract class CompareFilter implements FilterInterface
 {
-    /**
-     * @var ExpressionInterface|string
-     */
-    protected $column;
+    use ColumnFormatterTrait;
+
+    public static string $mainDateTimeFormat = 'Y-m-d H:i:s';
```

Comment:
Is that meant to be a constant?

Reply:
> Is that meant to be a constant?

No - that default format can be changed by user for all instances (in bootstrap.php). By default it most common format, but in my case i using pgsql timestamp_tz